packages/ui-test-locator/src/utils/locator.ts

```diff
@@ -30,11 +30,18 @@ import {
 } from '@instructure/ui-test-queries'
 import { SelectorOptions } from '@instructure/ui-test-queries/src/utils/selectors'
 import { QueryArguments } from '@instructure/ui-test-queries/src/utils/parseQueryArguments'
-import { GenericFunction } from '@instructure/ui-test-queries/src/utils/bindElementToMethods'
+import { QueriesHelpersEventsType } from '@instructure/ui-test-queries/src/utils/bindElementToUtilities'
 
-export function locator(
+// Cut the first argument of function that's an object's value
+type ObjWithCutFirstArg<T> = {
+  [K in keyof T]: T[K] extends (_: any, ...tail: infer TT) => infer R // if a function
+    ? (...args: TT) => R // cut first argument
+    : never // or use the original value
+}
```

Comment:
This type is kinda is re-declared in the file `bindElementsToMethods.ts`, I think we should just export it from one place and import the type where its needed.

Reply:
yep, fixed